Fehler beim Erstellen der Dokumentation

Beim Erstellen der Dokumentation tritt ein Fehler auf:

```
Exception occurred:
  File "~/.local/lib/python3.9/site-packages/docutils/parsers/rst/__init__.py", line 193, in parse
    inputstring, tab_width=document.settings.tab_width,
AttributeError: 'Values' object has no attribute 'tab_width'
The full traceback has been saved in /tmp/sphinx-err-srei8wfo.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:20: html] Fehler 2
```

--- a/README.md
+++ b/README.md
@@ -1,4 +1,4 @@
-# VisuAnalytics [![Tests](https://github.com/SWTP-SS20-Kammer-2/Data-Analytics/workflows/Automated%20Testing/badge.svg)](https://github.com/SWTP-SS20-Kammer-2/Data-Analytics/actions?query=workflow%3A%22Automated+Testing%22)
+# VisuAnalytics [![Tests](https://github.com/SWTP-SS20-Kammer-2/Data-Analytics/workflows/Automated%20Testing/badge.svg)](https://github.com/SWTP-SS20-Kammer-2/Data-Analytics/actions?query=workflow%3A%22Automated+Testing%22) [![Documentation Status](https://readthedocs.org/projects/visuanalytics/badge/?version=latest)](https://visuanalytics.readthedocs.io/de/latest/?badge=latest)
 
 ## Einleitung
 
@@ -29,12 +29,13 @@ _Benötigte Software_:
 
 - [Docker](https://www.docker.com/products/docker-desktop)
 
-Unter Windows muss ggf. noch ein Windows Subsystem for Linux installiert werden. 
+Unter Windows muss ggf. noch ein Windows Subsystem for Linux installiert werden.
 Weitere Informationen [hier](https://docs.docker.com/docker-for-windows/wsl/)
 
 _Docker-Container starten:_
 
 Die Pfade hinter `-v` müssen durch Pfade zu den Dateien, welche in [Konfiguration](#Konfiguration) beschrieben werden
+
 - bzw. durch den Pfad zum Output-Ordner - ersetzt werden.
 
 _Linux:_
@@ -59,7 +60,7 @@ docker run -t ^
 
 Falls man nur das Wordpress-Plugin verwenden will (siehe [hier](#wordpress-plugin-verwenden)) kann man auch
 eine Docker-Container starten, der das Frontend nicht enthält. Hierfür muss man nur anstelle von  
-`visuanalytics/visuanalytics`   
+`visuanalytics/visuanalytics`  
 im obigen Befehl  
 `visuanalytics/visuanalytics:latest-wordpress`
 eingeben.
@@ -321,14 +322,14 @@ _Benötigte Software_:
 
 - [Docker](https://www.docker.com/products/docker-desktop)
 
-Unter Windows muss ggf. noch ein Windows Subsystem for Linux installiert werden. 
+Unter Windows muss ggf. noch ein Windows Subsystem for Linux installiert werden.
 Weitere Informationen [hier](https://docs.docker.com/docker-for-windows/wsl/)
 
 _In den `src`-Ordner wechseln_: `cd src`
 
 _Tests ausführen_:
 
-- `docker-compose -f visuanalytics/docker-compose.test.yml up` 
+- `docker-compose -f visuanalytics/docker-compose.test.yml up`
 
 ### Ohne Docker
 

--- a/src/visuanalytics/requirements-dev.txt
+++ b/src/visuanalytics/requirements-dev.txt
@@ -3,5 +3,6 @@ sphinx==3.5.3
 sphinx_rtd_theme==0.5.1
 recommonmark==0.7.1
 sphinx-markdown-tables==0.0.15
+docutils==0.16
 
 -r requirements.txt

